twitter/scraper.py

```
@@ -59,7 +61,7 @@ def tweets_by_id(self, tweet_ids: list[int | str], **kwargs) -> list[dict]:
         @param kwargs: optional keyword arguments
         @return: list of tweet data as dicts
         """
-        return self._run(Operation.TweetResultByRestId, tweet_ids, **(latest_features | kwargs))
+        return self._run(Operation.TweetResultByRestId, tweet_ids, **(target_features | kwargs))
```

Ruff (0.15.0)

[error] `Operation` may be undefined, or defined from star imports

(F405)